qiskit/circuit/library/evolved_operator_ansatz.py

```diff
@@ -31,31 +39,28 @@ def __init__(
         evolution=None,
         insert_barriers: bool = False,
         name: str = "EvolvedOps",
-        parameter_prefix: Union[str, List[str]] = "t",
-        initial_state: Optional[QuantumCircuit] = None,
+        parameter_prefix: str | Sequence[str] = "t",
```

Comment:
would it make sense to revert this to??

```suggestion
        parameter_prefix: str | list[str] = "t",
```

Reply:
If we are expecting any tuples in this input, then this suggestion would probably not make sense, I also see the point of having a less restrictive typehint.

Reply:
I think we had `list` as catch-all for a container, so a Sequence should be fine 🙂